lib/fuzion/sys/internal_array.fz

```diff
@@ -55,3 +55,7 @@ module:public internal_array(T type, module data Pointer, public length i32) is
     #array.this[i] == o
   is
     setel T data i o
+
+  # NYI: a terrible hack to help air.Clazzes find fuzion.java.Java_Object
+  #
+  private bad fuzion.java.Java_Object := fuzion.java.Java_Object 0
```

Comment:
What code causes this not to be found? For most intrinsics, `air.Clazzes` should find this since they are declared as `intrinsic_contructor`. Is there some intrinsic that creates an array in a non-obivious way?
This should at least be documented better, maybe moved to `fuzion.java.fz`.